src/panels/config/dashboard/ha-config-dashboard.ts

```diff
@@ -134,45 +124,888 @@ class HaConfigDashboard extends LitElement {
+    const personData = await fetchPersons(this.hass!);
+
+    const personDataStorage = personData.storage.sort((ent1, ent2) =>
+      compare(ent1.name, ent2.name)
+    );
+    const personDataConfig = personData.config.sort((ent1, ent2) =>
+      compare(ent1.name, ent2.name)
+    );
+
+    this._persons = [...personDataStorage, ...personDataConfig];
+  }
+
+  private async _fetchIntegrationData() {
+    getConfigEntries(this.hass).then((configEntries) => {
+      this._configEntries = configEntries
+        .map(
+          (entry: ConfigEntry): ConfigEntryExtended => ({
+            ...entry,
+            localized_domain_name: domainToName(
+              this.hass.localize,
+              entry.domain
+            ),
+          })
+        )
+        .sort((conf1, conf2) =>
+          caseInsensitiveCompare(
+            conf1.localized_domain_name + conf1.title,
+            conf2.localized_domain_name + conf2.title
+          )
+        );
+    });
+  }
+
+  private _getEntities(configEntry: ConfigEntry): EntityRegistryEntry[] {
+    if (!this._entityRegistryEntries) {
+      return [];
+    }
+    return this._entityRegistryEntries.filter(
+      (entity) => entity.config_entry_id === configEntry.entry_id
+    );
+  }
+
+  private _getDevices(configEntry: ConfigEntry): DeviceRegistryEntry[] {
+    if (!this._deviceRegistryEntries) {
+      return [];
+    }
+    return this._deviceRegistryEntries.filter(
+      (device) =>
+        device.config_entries.includes(configEntry.entry_id) &&
+        device.entry_type !== "service"
+    );
+  }
+
+  private _getServices(configEntry: ConfigEntry): DeviceRegistryEntry[] {
+    if (!this._deviceRegistryEntries) {
+      return [];
+    }
+    return this._deviceRegistryEntries.filter(
+      (device) =>
+        device.config_entries.includes(configEntry.entry_id) &&
+        device.entry_type === "service"
+    );
+  }
+
+  private _getAutomations = memoizeOne(
+    (states: HassEntities): AutomationEntity[] => {
+      return Object.values(states)
+        .filter((entity) => computeStateDomain(entity) === "automation")
+        .slice(0, 2) as AutomationEntity[];
+    }
+  );
+
+  private _getScripts = memoizeOne(
+    (states: HassEntities): AutomationEntity[] => {
+      return Object.values(states)
+        .filter((entity) => computeStateDomain(entity) === "script")
+        .slice(0, 2) as AutomationEntity[];
+    }
+  );
+
+  private _getScenes = memoizeOne(
+    (states: HassEntities): AutomationEntity[] => {
+      return Object.values(states)
+        .filter((entity) => computeStateDomain(entity) === "scene")
+        .slice(0, 2) as AutomationEntity[];
+    }
+  );
+
+  private _getHelpers = memoizeOne(
+    (states: HassEntities): AutomationEntity[] => {
+      return Object.values(states)
+        .filter((entity) => HELPER_DOMAINS.includes(computeStateDomain(entity)))
+        .slice(0, 2) as AutomationEntity[];
+    }
+  );
+
+  private async _fetchTags() {
```

Comment:
```suggestion
  private async _getTags() {
```
maybe rename to make this consistent with _getScenes, _getScripts, and others